osu.Game.Rulesets.Taiko/Objects/Drawables/DrawableSwell.cs

```diff
@@ -248,28 +156,23 @@ protected override void CheckForResult(bool userTriggered, double timeOffset)
             }
         }
 
-        protected override void UpdateStartTimeStateTransforms()
-        {
-            base.UpdateStartTimeStateTransforms();
-
-            using (BeginDelayedSequence(-ring_appear_offset))
-                targetRing.ScaleTo(target_ring_scale, 400, Easing.OutQuint);
-        }
-
         protected override void UpdateHitStateTransforms(ArmedState state)
         {
-            const double transition_duration = 300;
-
             switch (state)
             {
                 case ArmedState.Idle:
-                    expandingRing.FadeTo(0);
+                    // Only for rewind support. Reallows user inputs if swell is rewound from being hit/missed to being idle.
+                    HandleUserInput = true;
```

Comment:
Is this state management still necessary? I did a few plays on my build with this line removed and everything seems to work normally.

Reply:
No, this isn't necessary anymore. This was apparently fixed separately after I created the original PR (see https://github.com/ppy/osu/pull/24196):

https://github.com/ppy/osu/blob/e794389fe83644323a563a343338e282783b53b1/osu.Game.Rulesets.Taiko/Objects/Drawables/DrawableSwell.cs#L204-L205

This also means the test I added back then has become redundant, you can remove it from this PR.